ui/entry-index.js

```diff
@@ -35,6 +35,7 @@ require('ngreact');
 require('jquery.scrollto');
 require('json-schema-defaults');
 require('./vendor/resizer.js');
+require('./vendor/jquery-migrate-3.0.0.js');
```

Comment:
This can be removed (the plugin is just to help with migration, it's not meant for long term use afaict)